scripts/build_releases.sh

```diff
@@ -62,32 +62,32 @@ if [ -n "$AUTH_HEADER" ]; then
 	GH_TAGS="${GH_API}/releases/tags/$RELEASE_TAG"
 
 	# release the version
-	response=$(curl  -sH "${AUTH_HEADER}" --data "{\"tag_name\":\"$RELEASE_TAG\", \"draft\":true, \"generate_release_notes\":true}" "$GH_API/releases")
+	response="$(curl  -sH "${AUTH_HEADER}" --data "{\"tag_name\":\"$RELEASE_TAG\", \"draft\":true, \"generate_release_notes\":true}" "$GH_API/releases")"
 
 	# extract id out of response
-	eval $(echo "$response" | grep -m 1 "id.:" | grep -w id | tr : = | tr -cd '[[:alnum:]]=')
-	[ "$id" ] || { echo "Error: Failed to get release id for tag: $tag"; echo "$response\n" >&2; exit 1; }
+	eval "$(echo "$response" | grep -m 1 "id.:" | grep -w id | tr : = | tr -cd '[[:alnum:]]=')"
+	[ "$id" ] || { echo "Error: Failed to get release id for tag: $GH_TAGS"; printf "%s\n" "$response" >&2; exit 1; }
```

Comment:
$tag should actually be replaced with $RELEASE_TAG . It's not critical in the sense that the script will break becaues this is just an echo command. $tag didn't exist previously as well (also in camino-node), which means that it was probably misreferenced here from the start. I suppose that the script is not really used for building releases as we are still probably doing it manually.